DDD Workshop.ipynb

```diff
@@ -0,0 +1,547 @@
+    "\n",
+    "# processing data into raw bar chart\n",
+    "rate_by_airport_time_raw = {}\n",
+    "for key, data_arr in data_by_airport_time_raw.items():\n",
+    "    occupancy_average = sum(data_arr) / len(data_arr)\n",
+    "    rate_by_airport_time_raw[key] = occupancy_average\n",
+    "\n",
+    "plt.figure(figsize=(17, 3))\n",
+    "names = rate_by_airport_time_raw.keys()\n",
+    "values = rate_by_airport_time_raw.values()\n",
+    "plt.ylabel('occupancy count')\n",
+    "plt.suptitle('Raw Airport Occupancy Rate')\n",
+    "plt.bar(names, values)\n",
+    "plt.show()\n",
+    "\n",
+    "\n",
+    "# processing data into cleaned bar chart\n",
+    "rate_by_airport_time_clean = {}\n",
+    "for key, data_arr in data_by_airport_time_clean.items():\n",
+    "    occupancy_average = sum(data_arr) / len(data_arr)\n",
+    "    rate_by_airport_time_clean[key] = occupancy_average\n",
+    "\n",
+    "\n",
+    "plt.figure(figsize=(17, 3))\n",
+    "names = rate_by_airport_time_clean.keys()\n",
+    "values = rate_by_airport_time_clean.values()\n",
+    "plt.bar(names, values, align='center')\n",
+    "plt.ylabel('occupancy percent')\n",
+    "plt.suptitle('Fixed Airport Occupancy Percentage')\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We see the issue with the original data set very clearly here, and we see that the conclusion from part 2 is still correct (JFK Redeye has the highest percentage), but we also see how small the percentage difference is.\n",
+    "\n",
+    "\n",
+    "There are many ways to communicate the conclusions that you have gathered, and there are many ways that those conclusions can be presented. We want to make sure that in our effort to convince, we're not stepping over the line of manipulating the other person. There are some heinous examples of misleading visualizations, some of the worst posted online and easy to search up in [aggreator lists](https://analythical.com/blog/examples-of-awful-data-visualization). It is worth taking some time to see the ways these can be misleading too, as the common mistakes can creep into your visualizations too if you're not careful. I've found [this guide from venngage](https://venngage.com/blog/misleading-graphs/) quite useful in familiarizing myself with the do not's of data viz.\n",
+    "\n",
+    "And finally, the complete guide, a bit old and extensive but still incredibly relevant as a resource: [The Quartz guide to bad data](https://qz.com/572338/the-quartz-guide-to-bad-data/). This piece is **HUGE**, but if you're looking for something and can't find it anywhere else, it likely lives in the tomes of Quartz.\n",
+    "\n",
+    "\n",
+    "## Resource and Reference Links\n",
+    "\n",
+    "I've aggregated the links to most resources throughout the article in order here, for reference, or for further exploration, which I would highly encourage for any aspiring data scientists or teachers among you.\n",
```

Comment:
I've aggregated the links to most resources throughout the article in order here for reference or for further exploration, which I would highly encourage for any aspiring data scientists or teachers among you.